Shradheya_Thakre/progress.md

```diff
@@ -1,78 +1,65 @@
 # Shradheya Thakre
 
 Overall Direction:
-- 
-- 
-- 
+- Reduce Technical Debt (Upgrade to Selenium 3.x)
+- Update test cases to optimize CI Build process
+- Add more test cases for missing features
+- Improving test infrastructure: Make it easier to write new tests, make tests less fragile, make test code easier to understand
+- Explore new JavaScript testing framework to replace QUnit
 
 ---
 
 Week 1 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Review PR #8813 [FeedbackRankQuestionUiTest.json: Fix data for Tests #8794 #8813](https://github.com/TEAMMATES/teammates/pull/8813)
+- [ ] Review PR #8770 [Add AdminExceptionTestActionTest #6902 #8770](https://github.com/TEAMMATES/teammates/pull/8770)
+- [ ] Resolve minor issues to improve testing E.g. [#8763](https://github.com/TEAMMATES/teammates/issues/8763) [#8768](https://github.com/TEAMMATES/teammates/issues/8768)
 
 Week 2 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Adding more Test cases based on `TODO` present in Testing code (Issues will be created accordingly)
+- [ ] Review PR's related to Testing which have been left untouched for a long time
 
 Week 3 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Start work on Upgrade to Selenium 3.x
+- [ ] Explore new Testing Framework for JS which can be useful for front end migration
 
 Week 4 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Continue work on Upgrade to Selenium 3.x
+- [ ] Work on improving readability of UI Tests (For e.g. the large amount of magic numbers)
 
 Week 5 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Refactor `FeedbackRubricQuestionUiTest` and `FeedbackRankQuestionUiTest`
+- [ ] Continue unfinished work from previous week
 
 Week 6 Plan:
-- [ ] 
-- [ ] 
-- [ ] 
+- [ ] Buffer week to complete unfinished work and investigate more on where testing needs to be improved
```

Comment:
I like that there is a buffer here. You can also add the investigation process in week 1, which I'm sure you will be doing consciously/subconsciously.